src/routes/generative-ai/+page.svelte

```diff
@@ -1,86 +1,258 @@
 <script lang="ts">
 	import LandingHero from '../components/landing-hero.svelte';
-	const title = 'Generative AI + ONNX Runtime';
+	import { fade } from 'svelte/transition';
+	import coffee from './coffee.png';
+	import genny from './genny.png';
+	import vision_ui from './vision_UI.png';
+	import mobile from './mobile.png';
+	import desktop from './desktop.png';
+	import cloud from './browser.png';
+	import aibrain from './aibrain.webp'
+	const title = 'Generative AI';
 	const description =
-		'Integrate the power of generative AI in your apps and services with ONNX Runtime. Broad platform support and deep optimizations empower usage of state-of-the-art models for image synthesis, text generation, and more.';
+		'Integrate the power of generative AI in your apps and services with ONNX Runtime. Broad platform support and deep optimizations empower usage of state-of-the-art models for text generation, audio synthesis, and more.';
 	const imgsrc = 'onnxruntimelogo';
 	const imgalt = 'ONNX Runtime Logo';
-	import stablediffusion1 from '../../images/StableDiffusion1.webp';
-	import stablediffusion2 from '../../images/StableDiffusion2.webp';
-	let image = 'https://i.ibb.co/0YBy62j/ORT-icon-for-light-bg.png'
-	let imageSquare = 'https://i.ibb.co/0YBy62j/ORT-icon-for-light-bg.png'
-	let authors = ['']
-	let keywords = 'onnxruntime, onnx runtime generative ai, onnx runtime generative ai models, onnx runtime generative ai deployment, onnx runtime generative ai performance, onnx runtime generative ai time to market, onnx runtime generative ai deploy anywhere, onnx runtime generative ai boost performance, onnx runtime generative ai improve time to market, onnx runtime generative ai production ready, onnx runtime generative ai lower latency, onnx runtime generative ai higher throughput, onnx runtime generative ai get innovations into production faster, onnx runtime generative ai testimonials, onnx runtime generative ai performance enhancements, onnx runtime generative ai production ready, onnx runtime generative ai lower latency, onnx runtime generative ai higher throughput, onnx runtime generative ai get innovations into production faster, onnx runtime generative ai testimonials, onnx runtime generative ai performance enhancements'
+	let image = 'https://i.ibb.co/0YBy62j/ORT-icon-for-light-bg.png';
+	let imageSquare = 'https://i.ibb.co/0YBy62j/ORT-icon-for-light-bg.png';
+	let authors = [''];
+	let keywords =
```

Comment:
I don't think we need meta keywords - not used by major search engines anymore.

Reply:
It's passed in by every page right now - you're def right that it's not super used, but I don't think it hurts 😅 If you'd like we could discuss refactoring the website outright though